app/controlplane/pkg/biz/casmapping.go

```diff
@@ -95,26 +100,42 @@ func (uc *CASMappingUseCase) FindCASMappingForDownloadByUser(ctx context.Context
 		return nil, NewErrInvalidUUID(err)
 	}
 
-	// Load organizations for the given user
-	memberships, err := uc.membershipRepo.FindByUser(ctx, userUUID)
+	// Load ALL memberships for the given user
+	memberships, err := uc.membershipRepo.ListAllByUser(ctx, userUUID)
```

Comment:
what if it's an owner? Do we have here the whole list of projects?

Reply:
No, but RBAC is only applied when the user is "member".